### PR TITLE
Added Documentation on Shipping UPS #7305

### PR DESCRIPTION
**Pull request by shaambhavipathak**

### CODE REVIEW

File: docs/create_new_shipment.rst [hunk 1]
```diff
@@ -0,0 +1,41 @@
+Create New Shipment
+====================
+
+1. Create a new Sale and fill in all the details.
```
Comment (rituparnapanda) [inline docs/create_new_shipment.rst]: new shipment

File: docs/create_new_shipment.rst [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+   When carrier is selected, another tab named UPS appears along with other
+   tabs. 
+
+.. figure:: images/1.png
+
+2. Click on ``UPS`` and select Carrier Details.
```
Comment (rituparnapanda) [inline docs/create_new_shipment.rst]: Click `USP` tab and select `Package Content Type` and `UPS Service Type`

File: docs/create_new_shipment.rst [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+.. figure:: images/2.png
+
+Once done, click on save.
+
+3. When the products have been assigned the carriers, they are ready to be
```
Comment (rituparnapanda) [inline docs/create_new_shipment.rst]: Once the shipment is assigned, it is ready for packing. Click `make Shipment` button to process.

File: docs/create_new_shipment.rst [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+.. figure:: images/2.png
+
+Once done, click on save.
+
+3. When the products have been assigned the carriers, they are ready to be 
+   processed. Click on ``Process``.
+
+.. figure:: images/3.png
+
+4. Once sale order is generated, it has to be assigned to shipment to be
```
Comment (rituparnapanda) [inline docs/create_new_shipment.rst]: Remove about sale, just write about shipment

File: docs/generate_shipping_label.rst [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+Shipping labels help in differentiating between different purchase orders.
+
+This documentation explains how shipping label can be generated for all
+shipments.
+
+.. note::
+
+   Shipping label can only be generated if the order is in the confirmed 
+   state. To create a new shipment, refer `Creating a New Shipment
+   <file:///home/shaambhavi/devel/trytond-shipping-ups/trytond-shipping-ups/build/create_new_shipment.html>`_.
```
Comment (rituparnapanda) [inline docs/generate_shipping_label.rst]: Link is from your local file, it wont work. And this is not the right way to interlink files within a doc.

File: docs/generate_shipping_label.rst [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+
+In the shipment tab, all the consignments which are marked as `Done` are
```
Comment (rituparnapanda) [inline docs/generate_shipping_label.rst]: Remove this line and continue taking an example of a shipment

File: docs/conf.py [hunk 7]
```diff
@@ -47,21 +48,25 @@
...
+copyright = u'2015, Openlabs'
+author = u'Openlabs'
 
 # The version info for the project you're documenting, acts as replacement for
 # |version| and |release|, also used in various other places throughout the
 # built documents.
 #
 # The short X.Y version.
-version = '1.0'
+version = 'VERSION'
```
Comment (sharoonthomas) [inline docs/conf.py]: I think the version and release should get the information from environment. See example here: 

https://github.com/openlabs/nereid/blob/develop/docs/conf.py#L56

File: docs/generate_shipping_label.rst [hunk 8]
```diff
@@ -0,0 +1,35 @@
+Generate Shipping Label
+========================
+
+Shipping labels help in differentiating between different purchase orders.
```
Comment (sharoonthomas) [inline docs/generate_shipping_label.rst]: what ??

File: docs/ups_configuration.rst [hunk 9]
```diff
@@ -0,0 +1,24 @@
+UPS Carrier Configuration
+=========================
+
+Before any shipment can be sent to the customer address, proper details about
```
Comment (sharoonthomas) [inline docs/ups_configuration.rst]: Before labels can be generated using this module, it has to be configured with your UPS account details.

File: docs/ups_configuration.rst [hunk 10]
```diff
@@ -0,0 +1,24 @@
...
+the carriers handling the parcels and shipments need to be attached with the 
+purchase order.
+
+The aim of this documentation is to help the user understand how to configure
```
Comment (sharoonthomas) [inline docs/ups_configuration.rst]: remove

File: docs/ups_configuration.rst [hunk 11]
```diff
@@ -0,0 +1,24 @@
...
+UPS Carrier for addition to purchase orders of customers. 
+
+1. Click on ``Inventory & Stock`` >> ``Configuration``.
```
Comment (sharoonthomas) [inline docs/ups_configuration.rst]: this is not correct. This information is now in the carrier itself.

File: docs/ups_configuration.rst [hunk 12]
```diff
@@ -0,0 +1,24 @@
...
+In the details window, double click on UPS Configuration.
+
+Fill in the details associated with UPS Account and then
+click on ``Save this record``.
+
+.. figure:: images/ups_configuration.png
+
+UPS Carrier configuration has been successfully done.
+
+
```
Comment (sharoonthomas) [inline docs/ups_configuration.rst]: this section should also explain how to obtain the API access key and related information. Shopify explains this https://docs.shopify.com/manual/settings/shipping/carrier-calculated-rates#xml-access-key

File: docs/create_new_shipment.rst [hunk 13]
```diff
@@ -0,0 +1,42 @@
...
+
+Once done, click on save.
+
+3. Once the shipment is assigned, it is ready for packing. Click
+``Make Shipment`` button to process.
+
+.. figure:: images/3.png
+
+4. Once shipment is generated, it has to be assigned to be
+shipped. Click on `Relate` and the click on `Shipments`.
```
Comment (rituparnapanda) [inline docs/create_new_shipment.rst]: Remove this line.

**Pull request closed without merging**